test/end-to-end/paths/probate/multipleExecutors.js

```diff
@@ -10,29 +10,22 @@ const optionNo = '-2';
 const bilingualGOP = false;
 const uploadingDocuments = false;
 
-// let grabIds;
-let stage1retries = -1;
-
 Feature('Multiple Executors flow').retry(TestConfigurator.getRetryFeatures());
 
 // eslint complains that the Before/After are not used but they are by codeceptjs
 // so we have to tell eslint to not validate these
 // eslint-disable-next-line no-undef
```

Comment:
can we remove this comment now?

Reply:
that comment is still present on the other tests like this from ‘GOP Intestacy spouse journey’, so I believe it's still needed, even though I’ve changed from `BeforeSuite` to `Before`.
I can take a look in a separate PR as to whether all the eslint test comments are still valid, but I think it's outside the scope of this change.